lib/spark_api/models/base.rb

```diff
@@ -139,6 +152,16 @@ def write_attribute(attribute, value)
         end
       end
 
+      private
+
+      def underscore(string)
+        string.to_s.gsub(/::/, '/').
+        gsub(/([A-Z]+)([A-Z][a-z])/,'\1_\2').
+        gsub(/([a-z\d])([A-Z])/,'\1_\2').
+        tr("-", "_").
+        downcase
+      end
```

Comment:
I was going to say we could use the `ActiveSupport` method for this, but this is literally the same method and saves us from bringing in that module. :+1: